### ISSUE TITLE
Error after manually revoking OAuth permissions

### ISSUE THREAD

**Issue by krlmlr** [hadley/httr#311]
This has occurred when using bigrquery, but I believe this to be an httr issue.

I revoked bigrquery permission for my Google account from https://myaccount.google.com/security#connectedapps. After this, each attempt to execute a query gives the following message (even after restarting R):

```
> query_exec(sql, project = project)
Auto-refreshing stale OAuth token.
Error in refresh_oauth2.0(self$endpoint, self$app, self$credentials) : 
  client error: (400) Bad Request
9: stop(http_condition(x, "error", call = sys.call(-1)))
8: stop_for_status(response)
7: refresh_oauth2.0(self$endpoint, self$app, self$credentials)
6: req$auth_token$refresh()
5: request_perform(req, hu$handle$handle)
4: POST(paste0(base_url, url), body = json, config(token = token), 
       add_headers(`Content-Type` = "application/json"), ..., query = prepare_bq_query(query)) at request.r#31
3: bq_post(url, body) at jobs.r#60
2: insert_query_job(query, project, destination_table = destination_table, 
       default_dataset = default_dataset) at query.r#32
1: query_exec(sql, project = project)
```

**Comment by nathangoulding** [comment]
@krlmlr Resolved with this: https://github.com/hadley/httr/pull/315.

**Comment by krlmlr** [comment]
@nathangoulding: Thanks, i'll take a look. FWIW, it's usually sufficient to `@`-ping the user and say "Fixes #311" or "Closes #311" in the pull request: https://help.github.com/articles/closing-issues-via-commit-messages/.

**Comment by nathangoulding** [comment]
Ahh, thanks.
